src/ServerSide/InternalTypes.elm

```diff
@@ -72,6 +72,9 @@ decodeNodeType =
                     "text" ->
                         Json.Decode.map TextTag (decodeTextTag)
 
+                    "keyed-node" ->
+                        Json.Decode.map NodeEntry (decodeNode)
```

Comment:
Decoding the keyed node children was wrong. Solution I have right now does some processing before stringifying... Not sure how I feel about it. Keeps the decoders tidy, but might be unexpected to have pre-stringing processing?